config/settings.yml

```diff
@@ -14,3 +14,100 @@ issue_manager:
   repo_names: []
 travis_event:
   enabled_repos: []
+trello:
+  # https://github.com/jeremytregunna/ruby-trello#configuration
+  oauth:
+    consumer_key:
+    consumer_secret:
+    oauth_token:
+    oauth_token_secret:
+  basic_auth:
+    developer_key:
+    member_token:
+  organization_id:
+  teams:
```

Comment:
The teams?  Or, the auth keys?

I sorta like having the teams in this file.  It makes it really easy to define the mappings of teams to boards and bug groups, etc... 

Maybe you mean something else?